cli/tox.ini

```diff
@@ -10,7 +10,7 @@ deps =
   -e..
 passenv = DCOS_* CI_FLAGS CLI_TEST_SSH_KEY_PATH CLI_TEST_MASTER_PROXY
 
-[testenv:py34-syntax]
```

Comment:
If you prefer, I think it is better to check with latest supported python version to catch `async` keyword used as a variable, for instance,

```python
def foo(async=True):
    pass
```
This snippet will pass with python3.4 but not with python3.5